src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -177,6 +178,38 @@ def __init__(self, arguments):
         self.model_specification = self.output_directory / 'model_specification.yaml'
 
 
+# TODO: The dumb thing for now. Can I make this work with NamedTuple ?
+class NativeSpecification:
+
+    def __init__(self, project, peak_memory, max_runtime, validation='n', threads=1, **__):
```

Comment:
Should validation be a boolean?

Reply:
it's actually a grid engine parameter here with more than two possibilities

Reply:
I figured something like that. If it moves to the CLI for initialization it can become a "choice"